src/status_im/ui/screens/keycard/subs.cljs

```diff
@@ -203,6 +203,5 @@
 (re-frame/reg-sub
  :keycard/frozen-card?
  (fn [db]
-   (let [{:keys [pin-retry-counter]}
-         (get-in db [:keycard :application-info])]
-     (zero? pin-retry-counter))))
+   (when (common/keycard-multiaccount? db)
+     (zero? (get-in db [:keycard :application-info :pin-retry-counter])))))
```

Comment:
its similar to `(and (common/keycard-multiaccount? db) (zero? (get-in db [:keycard :application-info :pin-retry-counter])))`

Reply:
thanks, indeed the version ``and`` makes more sense, fixed.